row-level-security/src/auth.js

```diff
@@ -0,0 +1,10 @@
+var cc = DataStudioApp.createCommunityConnector();
+
+// https://developers.google.com/datastudio/connector/reference#getauthtype
+function getAuthType() {
```

Comment:
getAuthType is already present in [main.js](https://github.com/googledatastudio/example-connectors/blob/master/row-level-security/src/main.js#L8). Either remove the function from main.js or remove this file.

Reply:
good catch, I'll remove the file.